### DISCUSSION TITLE
Multi currency

### DISCUSSION THREAD

**Discussion by rogerg1967** [openstore-ecommerce/OpenStore#244] [Q&A]
Is it possible to setup the price to display either £ or $ depending on the culture selected?

**Comment by leedavi** [comment]
It's possible in the template, to do a test and then display the correct format.

But, only 1 currency is supported.   So all figures would be the same.

Something like... (Of the top of my head, so it needs checking)

@if (Utils.GetCurrentCulture() == "en-US"
{
$
)
else
{
£
}

**Reply by rogerg1967** [reply]
Thank you for the answer, what we are looking to do is add a second price field which will hold and display the $ value and put a note to say that this is an approximate value in $ and all transactions are taken in £. This will at least give the customer an idea of local cost and keep the rest of the system working as normal.
Regards
Roger

**Reply by leedavi** [reply]
OK I see.  That should work.  Keep in mind the money conversions and textboxes tend to get awkward.
